x/oracle/keeper/tally_test.go

```diff
@@ -211,30 +211,30 @@ func (s *KeeperTestSuite) TestBuildClaimsMapAndTallyAboveThreshold() {
 	params.VoteThreshold = sdk.NewDecWithPrec(50, 2) // 50%
 	params.RewardBand = sdk.NewDecWithPrec(50, 2)    // 50%
 	params.AcceptList = types.DenomList{{
-		BaseDenom:   types.AtomDenom,
+		BaseDenom:   types.AtomSymbol,
 		SymbolDenom: types.AtomSymbol,
 		Exponent:    6,
 	}}
 	app.OracleKeeper.SetParams(ctx, params)
 
 	// initial exchange rate
-	app.OracleKeeper.SetExchangeRate(ctx, types.AtomDenom, sdk.MustNewDecFromStr("1.0"))
+	app.OracleKeeper.SetExchangeRate(ctx, types.AtomSymbol, sdk.MustNewDecFromStr("1.0"))
 
 	s.T().Log("TestBuildClaimsMapAndTally: >=50 votes out of 100 (above the threshold of 50%)")
 
 	const halfOfBondedValidatorsCount = 50
 
 	for valN := 0; valN < halfOfBondedValidatorsCount; valN++ {
 		app.OracleKeeper.SetAggregateExchangeRateVote(ctx, valAddresses[valN], types.NewAggregateExchangeRateVote([]types.ExchangeRateTuple{{
```

Comment:
Not sure if I understood you correctly @Tikaryan , using `Denom: types.AtomSymbol` breaks my brain there xD

Reply:
It's better to rename the parameter to `Symbol`.